core/ingest/src/main/java/mil/nga/giat/geowave/core/ingest/local/S3ParamsExtractor.java

```diff
@@ -0,0 +1,43 @@
+package mil.nga.giat.geowave.core.ingest.local;
```

Comment:
lets put these files associated with s3 URL handling in a separate mil.nga.giat.geowave.core.ingest.s3 package